_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+</body>
+```
+
+#### Passed Example 2
+
+The content in the column on the right is indicated with the word "right" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order.
+
+```html
+<head>
+<style>
+.col-container {
+  display: table;
+}
+.col {
+  display: table-cell;
+  padding: 16px;
+}
+</style>
+</head>
+<body>
+
+<div class="col-container">
+  <div class="col">
+    <p>Read the content in the right column (or below this paragraph) about the fuzzyness of cats</p>
+  </div>
+
+  <div class="col">
+    <p>Cats are usually fuzzy.</p>
+  </div>
+</div>
+
+</body>
+```
+
+#### Passed Example 3
+
+The content in the column on the right is a menu. The user is told to find the menu on the right (which is a visual indicator word) but the menu is also identified with the help of the "menu" heading above the content.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is no instruction.
+
+```html
+<body>
+	<p>In mathemathics a square is the result of multiplying a number by itself.</p>
+</body>
+```
+
+#### Passed Example 5
+
+The following text is tilted.
+
+```html
+<head>
+	<style> 
+		div.tilt {
+			height: 750px;
+			width: 150px;
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+			}
+	</style>
+</head>
+<body>
+	<div class="tilt">This text is tilted.</div>
+</body>
+```
+
+#### Passed Example 6
+
+The button is indicated by the word "round". But the word is also included in the text of the element.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+		border-radius: 50%;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 7
+
+The image is indicated by the word narrow but the alt text of the image also includes the word narrow.
```

Comment:
And I actually start to challenge the validity of this example… In regard to https://github.com/act-rules/act-rules.github.io/pull/832/files/f3d408896cc2a0b99546555face2ab629e780dca#r332439746 you say that the description including the visual word should be both visible and included in the accessibility tree, which is not the case for wide/narrow here… 😕

Reply:
I would say that the fact that the images are narrow and wide is visually communicated and communicated in the AT. So it would pass all SC correct?